src/internal/connector/exchange/api/mock/mail.go

```diff
@@ -1,36 +1,21 @@
 package mock
 
 import (
-	"github.com/alcionai/clues"
-
 	"github.com/alcionai/corso/src/internal/connector/exchange/api"
 	"github.com/alcionai/corso/src/internal/connector/graph"
 	"github.com/alcionai/corso/src/internal/connector/graph/mock"
 	"github.com/alcionai/corso/src/pkg/account"
 )
 
-func NewService(creds account.M365Config, opts ...graph.Option) (*graph.Service, error) {
```

Comment:
This was a new version of [`api.NewService`](https://github.com/alcionai/corso/blob/3e0f81dddbf66a7ef5d8901f21d2ed1676fe16dd/src/internal/connector/exchange/api/api.go#L91) and that is why I left it here instead of moving to `graph`

Reply:
That may be the intent, but the code doesn't follow up.  What's implemented here is simply a graph service.  For that matter, now that I'm looking at it again, it isn't even a mock. It's just a regular graph.Service.